main.tf

```diff
@@ -566,7 +584,7 @@ data "aws_iam_policy_document" "ecs_task_access_secrets_with_kms" {
 }
 
 resource "aws_iam_role_policy" "ecs_task_access_secrets" {
-  count = var.atlantis_github_user_token != "" || var.atlantis_gitlab_user_token != "" || var.atlantis_bitbucket_user_token != "" ? 1 : 0
+  count = var.atlantis_github_user_token != "" || var.atlantis_github_app_key != "" || var.atlantis_gitlab_user_token != "" || var.atlantis_bitbucket_user_token != "" ? 1 : 0
```

Comment:
```suggestion
  count = local.has_secrets ? 1 : 0
```

Reply:
I tested this suggestion and it ran fine. The change is included in the "code improvements" commit.